### PR TITLE
Restore XML format to WIF configuration section

### PR DESCRIPTION
**Pull request by nemrism**
## Summary

Apply XML formatting and link to other pages of configuration section

Contributes to #2038

### CODE REVIEW

**Review by rpetrusha — APPROVED** [review]
Thank you, @nemrism, for another PR that improves the formatting of the configuration hierarchy. I'll merge your PR now.

File: docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ author: "BrucePerlerMS"
 # \<add>
 Adds the specified security token handler to the token handler collection.  
   
- \<system.identityModel>  
-\<identityConfiguration>  
-\<securityTokenHandlers>  
-\<add>  
+[**\<configuration>**](../configuration-element.md)\
```
Comment (gewarren) [inline docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md]: Shouldn't we just put these in xml code blocks? @mairaw

Comment (mairaw) [reply docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md]: they link to the specific elements, so we can't put them in xml code blocks

Comment (nemrism) [reply docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md]: Let's debate on this again? 😁

Comment (nemrism) [reply docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md]: Let's debate on this again? 😁

**Pull request merged**